packages/arepa-wallet/package.json

```
@@ -0,0 +1,27 @@
+{
+  "name": "@near-wallet-selector/arepa-wallet",
+  "version": "8.9.7",
+  "description": "Arepa Wallet package for NEAR Wallet Selector.",
+  "keywords": [
+    "near",
+    "blockchain",
+    "wallets",
+    "dapps",
+    "near-protocol",
+    "near-blockchain",
+    "wallet selector",
+    "browser wallet",
+    "arepa Wallet"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/near/wallet-selector.git"
+  },
+  "bugs": {
+    "url": "https://github.com/near/wallet-selector/issues"
+  },
+  "homepage": "https://github.com/near/wallet-selector/tree/main/packages/arepa-wallet",
+  "peerDependencies": {
+    "near-api-js": "^1.0.0 || ^2.0.0"
```

Can we use latest near-api-js v4?

---

I can't see the changes ... can you double check?
```suggestion
    "near-api-js": "^1.0.0 || ^2.0.0"
```
```suggestion
    "near-api-js": "^1.0.0 || ^2.0.0"
```

---

Ok, we were executing the commit against the source code, sorry for that, can you verify now?